src/validation_workflow/validation_args.py

```diff
@@ -123,22 +123,30 @@ def __init__(self) -> None:
             "--validate-digest-only",
             action="store_true",
             default=False,
-            help="(optional) Validate digest only; will not run docker to test API")
+            help="(optional) Validate digest only; will not run docker to test API",
+            dest="validate_digest_only")
         group.add_argument(
             "--using-staging-artifact-only",
             action="store_true",
             default=False,
-            help="(optional) Use only staging artifact to run docker and API test, will not validate digest")
+            help="(optional) Use only staging artifact to run docker and API test, will not validate digest",
+            dest="using_staging_artifact_only")
```

Comment:
Why?
```
dest="using_staging_artifact_only")
```

Reply:
I have just added it for easy readability, as the value using-staging-artifact-only is using '-'.

Reply:
I mean to say why it ended up with `)`? For both dest you added there?

Reply:
help="(optional) Use only staging artifact to run docker and API test, will not validate digest")  `)` comes from here 
            help="(optional) Use only staging artifact to run docker and API test, will not validate digest",
            dest="using_staging_artifact_only")

Reply:
Then correct the location of the `)`?